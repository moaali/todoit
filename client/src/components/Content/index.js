import React from 'react'
import PropTypes from 'prop-types'
import { Sidebar, Main, Offcanvas, AddButton } from 'components';

const Content = ({ className, ...restProps }) => (
  <section className={`peers fxw-nw content ${className}`} {...restProps}>
    <div className="peer">
      <Sidebar />
    </div>
    <div className="peer peer-greed">
      <Main />
    </div>
    <div className="peer">
      <Offcanvas />
    </div>
    <AddButton />
  </section>
);

Content.defaultProps = {
  className: '',
}

Content.propTypes = {
  className: PropTypes.string,
}
export default Content;
