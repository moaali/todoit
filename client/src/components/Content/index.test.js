import React from 'react';

import Content from './';

describe('Content', () => {
  it('should match snapshot', () => {
    const contentElement = shallow(<Content />);
    expect(toJson(contentElement)).toMatchSnapshot();
  });
})
