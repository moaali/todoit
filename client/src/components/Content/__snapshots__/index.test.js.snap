// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should match snapshot 1`] = `
<section
  className="peers fxw-nw content "
>
  <div
    className="peer"
  >
    <Sidebar />
  </div>
  <div
    className="peer peer-greed"
  >
    <Main />
  </div>
  <div
    className="peer"
  >
    <inject-Offcanvas-with-store />
  </div>
  <inject-AddButton-with-store />
</section>
`;
